recipes/esys-escript/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -x -e
+set -o pipefail
+
+CXXFLAGS="${CXXFLAGS} -fPIC -w -fopenmp"
+
+cd ${SRC_DIR}/escript
+scons -j"${CPU_COUNT}" \
+    options_file="${SRC_DIR}/escript/scons/templates/stretch_options.py" \
+    prefix=${PREFIX} \
+    build_dir=${BUILD_PREFIX}/escript_build \
+    boost_prefix=${PREFIX} \
+    boost_libs="boost_python27" \
+    cxx=${CXX} \
+    cxx_extra="-w -fPIC" \
+    ld_extra="-L${LIBRARY_PATH} -lgomp" \
```

Comment:
Should this be `-L${PREFIX}/lib`? I'm not sure why libgomp.so is not being linked in

Reply:
I thought that `LIBRARY_PATH` was the same, but maybe I'm wrong. I've just changed it.